### PR TITLE
add scope tags and aria-labels to instance table

### PR DESCRIPTION
**Pull request by Pandoraaa**
This PR will improve a11y with a table caption, table structure with scope to help screen readers and aria-label for its yes/no icons.

### CODE REVIEW

File: client/src/app/shared/instance/instance-features-table.component.scss [hunk 1]
```diff
@@ -27,6 +28,13 @@ table {
   td {
     vertical-align: middle;
   }
+
+  caption {
+    caption-side: top;
+    font-size: 16px;
+    font-weight: $font-semibold;
```
Comment (rigelk) [inline client/src/app/shared/instance/instance-features-table.component.scss:35]:
```suggestion
    font-size: 15px;
    font-weight: $font-bold;
```

Comment (rigelk) [reply client/src/app/shared/instance/instance-features-table.component.scss:35]: This is to provide the same style as `label` elements, but you might find it too bold - which do you prefer?

Comment (Pandoraaa) [reply client/src/app/shared/instance/instance-features-table.component.scss:35]: I find the $font-bold too bold 😄 
I'll change the font-size :)

**Comment by rigelk** [comment]
Thanks again @Pandoraaa :slightly_smiling_face:

**Pull request merged**